tests/test_modules.py

```diff
@@ -594,3 +596,28 @@ def test_self_mutating1(self):
         self.assertTrue(torchdynamo.testing.same(out2, out3))
         self.assertTrue(torchdynamo.testing.same(out2, out4))
         self.assertEqual(cnt.frame_count, 3)
+
+    def test_simple_torch_function(self):
+        def my_compiler(gm: torch.fx.GraphModule, example_inputs: List[torch.Tensor]):
```

Comment:
For unittests I'd suggest using `CompilerCounter` and asserting something about the number of captured ops.  See above for examples.